src/main.md

```diff
@@ -295,9 +295,12 @@ memory (Flash); the LMA is where in Flash those initial values are stored.
 Finally, we associate a symbol to the LMA of `.data`.
 
 On the Rust side, we zero the `.bss` section and initialize the `.data` section. We can reference
-the symbols we created in the linker script from the Rust code. The *addresses* of these symbols are
+the symbols we created in the linker script from the Rust code. The *addresses*[^1] of these symbols are
 the boundaries of the `.bss` and `.data` sections.
 
+[^1]: The fact that the addresses of the linker script symbols must be used here can be confusing and
```

Comment:
This does get render as a footnote by mdbook but it's not automatically placed at the end of the page (see image below). You'll have to move it to the bottom manually.

---

![20180808-105840](https://user-images.githubusercontent.com/5018213/43849141-55214d30-9afa-11e8-91b9-45bc0e7964a7.png)